__tests__/pages/client/index.test.tsx

```diff
@@ -24,16 +47,12 @@ describe('Client', () => {
       value: { search: '?ticketNo=t123' },
     });
     const tixSpy = jest.spyOn(clientService, 'getTicket')
```

Comment:
Since the service makes live network calls to the DB, and we might want to run tests in an isolated environment, I would suggest mocking the functionality of service calls instead of spying on them and allowing the actual code to run.

Reply:
done